configure

```diff
@@ -1535,13 +1535,6 @@ case "${ARCH}" in
 
             check_sse41_intrinsics
 
-            if test ${HAVE_SSE41_INTRIN} -eq 1; then
-                CFLAGS="${CFLAGS} -DX86_SSE41_ADLER32"
-                SFLAGS="${SFLAGS} -DX86_SSE41_ADLER32"
-                ARCH_STATIC_OBJS="${ARCH_STATIC_OBJS} adler32_sse41.o"
-                ARCH_SHARED_OBJS="${ARCH_SHARED_OBJS} adler32_sse41.lo"
```

Comment:
Just replace so that only `-DX86_SSE41` is added, so we don't need to remove the whole if block and re-add it later... This makes it more consistent with how we do checks with cmake, and line `check_sse41_intrinsics` doesn't become stray...